markdown/articles/2020/03/22/DDFAQs.md

```diff
@@ -0,0 +1,600 @@
+main mana enablers that can be found across all variants. Dark Ritual especially allows
+you to cast Doomsday off of a single B source which is great against mana denial strategies
+many decks deploy.
+
+It wouldn't be a post-2019 article without mentioning {{Arcum's Astrolabe}}. By manipulating
+the manabase so it includes a heavy Snow-Covered basic count you can include this 
+efficient {{Prophetic Prism}} that enables mana fixing and does not lose card advantage
+whilst playing it. 
+
+You may note I have not yet discussed {{Lion's Eye Diamond}} here, this will be covered in 
+"Combo Pieces".
+
+**DDFT and DDEFT
+
+For these variants both {{Dark Ritual}} and {{Lotus Petal}} should be considered as 4-ofs.
+In addition some consideration might be given to running additional acceleration in the form
+of up to 1 {{Cabal Ritual}} or {{Rain of Filth}}. These variants want to go fast and as such,
+the levels of fast mana should be increased to facilitate this. Conversely, these decks
+most likely do not want to run Astrolabe as it promotes a slower gameplay style overall but
+can still reasonably incorporate it if desired.
+
+**Meandeck
+
+Meandeck lists are less concerned with going fast. They still want to run between 3-4 Rituals
+as it enables the more mana efficient "kill turns" however they likely only want between 1-3
+Petals, normally around 2 at most. This is the variant that most favours the Astrolabe
+fixing plan being able to drop basics at a steady rate and favouring resilience over speed within
+the gameplan. If you are running Astrolabe it should be between 2-4 however higher numbers are 
+recommended.
+
+
+### 2.3 Cantrips
+
+Cantrips are a staple of almost any Legacy deck that runs U and this one is no exception.
+Just like other combo variants, Doomsday wants to find the pieces it needs as quickly and reliably
+as it can so it tends to have a greater number of cantrips relative to other "fairer" decks.
+Unlike other decks like TES however, the cantrips can actually form an integral part of the
+combo itself and so ensuring these are at a maximum is key. You don't want to be having to
+"burn" through cantrips to find pieces to then find yourself unable to draw into an otherwise
+game-winning pile.
+
+As a result I would expect any Doomsday deck to be running {{Brainstorm}}, {{Ponder}} and {{Preordain}}
+in various ratios of 4:4:X. In addition to this some lists will be running the previously cited {{Arcum's Astrolabe}}
+along with some spicier elements such as {{Street Wraith}} and {{Edge of Autumn}} however more on those later.
+Combine these with some of the cantripping protection elements and more niche "combo cards" like
+{{Conjurer's Bauble}} you should end up with a very high cantrip number.
+
+**DDFT
+
+I would expect a modern DDFT list to run the default 8 of Ponder x Brainstorm and anywhere between
+2 and 4 copies of Preordain. I would also not be surprised to expect maybe 0-2 Street Wraith. If
```

Comment:
Maybe say something about how Preordain tends to be fewer in number the faster the deck wants to be? I was on 0 Pre with full blown Echo Doomsday, but I've also recently been testing 1 Echo with the full 4 Preordain and fewer Rits. Not a required change, just a thought.

Reply:
Added comment.